packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -41,10 +41,11 @@ let globalCooldownTimeout = null;
  * methods to toggle this state. Ensures only one tooltip is open at a time and controls
  * the delay for showing a tooltip.
  */
-export function useTooltipTriggerState(props: TooltipTriggerProps): TooltipTriggerState {
+export function useTooltipTriggerState(props: TooltipTriggerProps = {}): TooltipTriggerState {
```

Comment:
all options are optional so IMHO it makes sense to make `props` optional as well